vast.conf

```
@@ -1,116 +1,380 @@
+; This is an example configuration file for VAST, striving to show all available
+; options. Lines starting with a semicolon are commented out. Options in angle
+; brackets have their default value determined at runtime.
+
+; Options that apply to VAST.
 system {
-;; The host and port to listen at and connect to.
-; endpoint = "localhost:42000"
+  ; The host and port to listen at and connect to.
+  ;endpoint = "localhost:42000"
+
+  ; The file system path used for persistent state.
+  ;db-directory = "vast.db/"
+
+  ; The file system path used for log files.
+  ;log-file = <server.log in db-directory>
```

```suggestion
  ;log-file = "${db-directory}/server.log"
```

This seems a bit less ambiguous IMO. Maybe `<db-directory>/server.log`, but I would not combine text and variable in that fashion.

---

The angle brackets are consistent with the style used by CAF for values that are determined at runtime. The comment at the very top of the file explains what the angle brackets mean.

---

Okay, but is `in` a keyword? that confuses me. Having a `/` would make things more readable IMO.

---

As in `<db-directory/server.log>`?

---

```suggestion
  ;log-file = "<db-directory>/server.log"
```